src/modules/dashboard/components/SuggestionsListItem/SuggestionsListItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Heading from '~core/Heading';
+import { OneSuggestion } from '~data/index';
+
+import { getFriendlyName } from '../../../users/transformers';
+
+import styles from './SuggestionsListItem.css';
+
+const MSG = defineMessages({
+  byAuthorText: {
+    id: 'Dashboard.SuggestionsListItem.byAuthorText',
+    defaultMessage: 'by {creator}',
+  },
+});
+
+interface Props {
+  suggestion: OneSuggestion;
```

Comment:
Why is this called "one" suggestion ? :)

I'm assuming it's referring to a singular suggestion, but then why not use the "suggestion/suggestions" pair ?

Yeah, yeah, I know... this is nit-picking... :)

Reply:
`type Suggestion` **is** indeed created by the code generator, but I'm using `OneSuggestion` to describe  a single suggestion from a query, considering the fact that we might have multiple suggestion queries in the future.

It basically gives us the freedom to say something along the lines of:

```typescript
type OneSuggestion =
  | ColonySuggestionsQuery['colony']['suggestions'][number]
  | UserSuggestionsQuery['user']['suggestions'][number];
```